container/pom.xml

```diff
@@ -45,10 +45,6 @@
           <groupId>io.airlift</groupId>
           <artifactId>airline</artifactId>
         </exclusion>
-         <exclusion>
```

Comment:
Please run 'mvn dependency:tree' before and after to check for any changes.

Reply:
Nothing there.